packages/views/order_edit.py

```diff
@@ -0,0 +1,191 @@
+from django.core import serializers
+from packages.models import Address, City, Order, Package, Partner, Street
+from django.shortcuts import render
+from util import is_member, is_staff
+from packages.util import string_data_lists_context
+from django.http import HttpResponse
+from django.template import loader
+
+def order_edit_view(request, partner_name, order_id):
+    partner = Partner.objects.get(name=partner_name)
+    order = Order.objects.get(id=order_id)
+
+    if not is_member(request.user, partner.name) and not is_staff(request.user):
+        return render(request, 'errors/access_restricted.html', {})

```

Comment:
The `access_restricted.html` template is broken. It uses the old `block` names.

Reply:
forgot to implement it oops